src/routes/amt/getVersion.test.ts

```diff
@@ -2,102 +2,12 @@ import { version } from './getVersion'
-      IsEntity: 'true',
-      VersionString: '15.0.23'
-    }, {
-      InstanceID: 'Netstack',
-      IsEntity: 'true',
-      VersionString: '15.0.23'
-    }, {
-      InstanceID: 'AMTApps',
-      IsEntity: 'true',
-      VersionString: '15.0.23'
-    }, {
-      InstanceID: 'AMT',
-      IsEntity: 'true',
-      VersionString: '15.0.23'
-    }, {
-      InstanceID: 'Sku',
-      IsEntity: 'true',
-      VersionString: '16392'
-    }, {
-      InstanceID: 'VendorID',
-      IsEntity: 'true',
-      VersionString: '8086'
-    }, {
-      InstanceID: 'Build Number',
-      IsEntity: 'true',
-      VersionString: '1706'
-    }, {
-      InstanceID: 'Recovery Version',
-      IsEntity: 'true',
-      VersionString: '15.0.23'
-    }, {
-      InstanceID: 'Recovery Build Num',
-      IsEntity: 'true',
-      VersionString: '1706'
-    }, {
-      InstanceID: 'Legacy Mode',
-      IsEntity: 'true',
-      VersionString: 'False'
-    }, {
-      InstanceID: 'AMT FW Core Version',
-      IsEntity: 'true',
-      VersionString: '15.0.23'
-    }],
-    status: 200
-  },
-  AMT_SetupAndConfigurationService: {
-    response: {
-      CreationClassName: 'AMT_SetupAndConfigurationService',
-      ElementName: 'Intel(r) AMT Setup and Configuration Service',
-      EnabledState: '5',
-      Name: 'Intel(r) AMT Setup and Configuration Service',
-      PasswordModel: '1',
-      ProvisioningMode: '1',
-      ProvisioningServerOTP: 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=',
-      ProvisioningState: '2',
-      RequestedState: '12',
-      SystemCreationClassName: 'CIM_ComputerSystem',
-      SystemName: 'Intel(r) AMT',
-      ZeroTouchConfigurationEnabled: 'true'
-    },
-    responses: {
-      Header: {
-        To: 'http://schemas.xmlsoap.org/ws/2004/08/addressing/role/anonymous',
-        RelatesTo: '2',
-        Action: 'http://schemas.xmlsoap.org/ws/2004/09/transfer/GetResponse',
-        MessageID: 'uuid:00000000-8086-8086-8086-000000000003',
-        ResourceURI: 'http://intel.com/wbem/wscim/1/amt-schema/1/AMT_SetupAndConfigurationService'
-      },
-      Body: {
-        CreationClassName: 'AMT_SetupAndConfigurationService',
-        ElementName: 'Intel(r) AMT Setup and Configuration Service',
-        EnabledState: '5',
-        Name: 'Intel(r) AMT Setup and Configuration Service',
-        PasswordModel: '1',
-        ProvisioningMode: '1',
-        ProvisioningServerOTP: 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=',
-        ProvisioningState: '2',
-        RequestedState: '12',
-        SystemCreationClassName: 'CIM_ComputerSystem',
-        SystemName: 'Intel(r) AMT',
-        ZeroTouchConfigurationEnabled: 'true'
-      }
-    },
-    status: 200
-  }
-}
+import { setupAndConfigurationServiceResponse, softwareIdentityResponse, Versionresponse } from '../../test/helper/wsmanResponses'
 
 describe('version', () => {
   let resSpy
   let req
-  let versionSpy
+  let setupAndConfigurationServiceSpy, softwareIdentitySpy
```

Comment:
use new line for declaring each variable also should type the spys with jest.SpyInstance

Reply:
done